libpod/runtime_img.go

```diff
@@ -370,8 +370,7 @@ func (k *Image) GetLocalImageName() (string, error) {
 			}
 		}
 	}
-	fqname, _ := k.GetFQName()
-	return fqname, nil
+	return "", errors.Errorf("unable to find image locally")
```

Comment:
Should we return 

errors.Wrapf(storage.ErrImageUnknown, "unable to find image locally")

Also you should probably be consistent on the error message.  Above you obtain, here you find.  I think we should just stick to find.

Reply:
done on both